demo/test_sequencer.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+L2BINARYNAME=rollupd
+ROLLUP_NAME=test-rollup
+GOPATH=$HOME/go
+L1_CHAIN_ID=sim
+L2_CHAIN_ID=sim2
+L1_KEYRING_DIR=~/.simapp
+L2_KEYRING_DIR=~/.l2simapp
```

Comment:
Where this directory is made?

Reply:
here
https://github.com/Finschia/finschia-sdk/blob/for-ramus-sdk/init_node.sh#L75